src/Packagist/WebBundle/Resources/public/css/main.css

```diff
@@ -1003,10 +1003,10 @@ ul.packages .metadata-block:last-child {
 .package-aside .details {
   padding-left: 0;
 }
-.package .package-aside i, .package .readme-expander i {
+.package .package-aside i:not(.fa), .package .readme-expander i:not(.fa) {
```

Comment:
Not sure of what you mean. This?

```css
i:not(.fa):not(.glyphicon)
```

Or maybe you want me to add a `i.fa` rule to rollback to the default color? It's a little bit to tricky IMHO.